test/c_api_test/test.c

```diff
@@ -15,8 +15,8 @@
 #include "irods/dataObjClose.h"
 #include "irods/checksum.h"
 
-#if IRODS_VERSION_INTEGER != 4003001
-    #error "IRODS_VERSION_INTEGER needs attention"
+#if IRODS_VERSION_INTEGER != 4090000
+#  error "IRODS_VERSION_INTEGER needs attention"
```

Comment:
is this edit correct?

Reply:
Yes

Reply:
sorry, i meant the placement of the # sign

Reply:
Oh, I wasn't planning on including that change originally, but kory asked me to include the cmake-format suggestions.

Reply:
This kind of thing bit me with the OpenSSL API compatibility flag, so I wanna make sure I understand :) https://www.openssl.org/docs/manmaster/man7/OPENSSL_API_COMPAT.html

So it's... major * 1000000 + minor * 1000 + patch ?

4.90.0 => 4 * 1000000 + 90 * 1000 + 0 = 4000000 + 90000 + 0 = 4090000
4.3.1 => 4 * 1000000 + 3 * 1000 + 1 = 4000000 + 3000 + 1 = 4003001

Reply:
> So it's... major * 1000000 + minor * 1000 + patch ?

Correct. It's defined here: https://github.com/irods/irods/blob/7a8a88c022d63a23036622d468f6bee91817d97f/lib/core/include/irods/irods_version.h.in#L8

Reply:
Ah, very good. Thanks